### ISSUE TITLE
Error: Call to undefined method Facebook\Webdriver\Remote\RemoteWebDriver::getCapabilities()

### ISSUE THREAD

**Issue by zoispag** [laravel/dusk#464]
Since the Chrome update to v64 this week, my test that are uploading a file as part of a modal, are failing.
Tests are running through TeamCity on ubuntu 16.04

I've tried several ChromeDriver version including 2.34 & 2.35

Anyone experiencing something similar?

**Comment by zoispag** [comment]
This comes from the **attach** method in InteractsWithElements trait:

```php
    /**
     * Attach the given file to the field.
     *
     * @param  string  $field
     * @param  string  $path
     * @return $this
     */
    public function attach($field, $path)
    {
        $element = $this->resolver->resolveForAttachment($field);

        if ($this->driver->getCapabilities()->getBrowserName() == 'phantomjs') {
	        $element->setFileDetector(new UselessFileDetector())->sendKeys($path);
        } else {
	        $element->setFileDetector(new LocalFileDetector)->sendKeys($path);
        }

        return $this;
    }
```
to check if the browser is phantomjs.

**Comment by oncet** [comment]
I'm having the same error while trying the example test provided by Dusk 2.0.12.

```
namespace Tests\Browser;

use Tests\DuskTestCase;
use Laravel\Dusk\Browser;
use Illuminate\Foundation\Testing\DatabaseMigrations;

class ExampleTest extends DuskTestCase
{
    /**
     * A basic browser test example.
     *
     * @return void
     */
    public function testBasicExample()
    {
        $this->browse(function (Browser $browser) {
            $browser->visit('/')
                    ->assertSee('Laravel');
        });
    }
}
```

After running `php artisan dusk` I get:

```
1) Tests\Browser\ExampleTest::testBasicExample
Error: Call to undefined method Facebook\WebDriver\Remote\RemoteWebDriver::getCapabilities()
```
I'm using Laravel 5.5.34.

**Comment by oncet** [comment]
All right, fixed after updating the facebook/webdriver package.

```
composer update facebook/webdriver
  - Updating facebook/webdriver (1.1.1 => 1.5.0): Downloading (100%)
```
`php artisan dusk` no longer throws the error and all is green.

**Comment by pacoorozco** [comment]
I've the same problem, in order to reproduce it:
1. Install laravel and copy `.env.example`as `.env`
1. Run `composer require --dev laravel/dusk`
1. Run `php artisan dusk:install`
1. Run `php artisan key:generate`

And finally run `php artisan dusk` and the error will appear:

```
1) Tests\Browser\ExampleTest::testBasicExample
Error: Call to undefined method Facebook\WebDriver\Remote\RemoteWebDriver::getCapabilities()
```

**Comment by pacoorozco** [comment]
It's **solved** see #385

**Comment by zoispag** [comment]
Solved by updating Dusk months ago.
It seems that the tickets are not monitored here! 😞
